From 51fa4502039ec09d4a9e1d6b2fa73e5a1f789f94 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 15 Jun 2021 08:31:06 -0400
Subject: [PATCH 1/1] xcas.desktop: fix "Version" field.

The "Version" field in a desktop file refers to the version of the
specification, not the version of the software it launches:

  https://specifications.freedesktop.org/desktop-entry-spec

This issue has already been fixed upstream:

  https://xcas.univ-grenoble-alpes.fr/forum/viewtopic.php?f=3&t=2716

and will be part of the next release.
---
 xcas.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xcas.desktop b/xcas.desktop
index 9e1b648..fad99cd 100644
--- a/xcas.desktop
+++ b/xcas.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=1.7
+Version=1.0
 Keywords=computer algebra system;CAS;
 Keywords[fr]=calcul formel;CAS;
 Name=Xcas
-- 
2.31.1

